tests/AlgoliaSyncEngine.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
Let's discuss about this.

Reply:
as discussed together we will keep it that way